release-canary.js

```diff
@@ -0,0 +1,85 @@
+const semver = require('semver');
+const fs = require('fs');
+const childProcess = require('child_process');
+const wd = process.cwd();
+const gitRevision = childProcess
+  .execSync('git rev-parse --short HEAD')
+  .toString()
+  .trim();
+
+function exec(command) {
+  try {
+    console.log(`Executing: ${command}`);
+    childProcess.execSync(command);
+  } catch (error) {
+    console.log(`Error while executing ${command}\n` + error);
+    throw new Error();
```

Comment:
would be nice with some message here, or rethrowing the old error (i.e. `throw error`